<?xml version="1.0"?>
<launch>
    <arg name="model_path" default="/home/iiwa-kuka/software/russell_ws/src/flowbot3d/models/camera_frame_mask/epoch=99-step=78600.ckpt"/>
    <node pkg="flowbot3d_ros" name="flowbot3d_ros" type="flowbot3d_ros.py" args="--model_path $(arg model_path)" output="screen">
    <remap from="/input/depth_image" to="/live_camera/aligned_depth_to_color/image_raw"/>
    <remap from="/input/mask" to="/rqt_image_segmentation/mask"/>
    </node>
</launch>
